src/utils/index.js

```diff
@@ -7,6 +7,7 @@ var objectAssign = require('object-assign');
 
 var warn = debug('utils:warn');
 
+module.exports.Timer = require('./timer');
```

Comment:
@dmarcos this was my first try, but the current implementation it's quite ugly (https://github.com/mrdoob/three.js/blob/master/src/core/Clock.js) you need to call `getElapsedTime()` and `getDelta()` but you'll get different values as it will call `performance.now()` again. Or you call `getDelta()` and then access `elapsedTime` that doesn't seems very coherent.
Also it works with seconds instead of ms, so we need to multiply everything by 1000.
I would better stick for now with our implementation and probably open a PR in three.js but I guess it's widely used so it will break many things if we change the implementation a lot.